ksp_plugin_adapter/ksp_plugin_adapter_linux.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the purpose of this file?  Is it that you need a specific build script for use with Mono?  What's different from the vanilla `ksp_plugin_adapter.csproj`?